testkit/src/main/scala/org/bitcoins/testkit/core/gen/TLVGen.scala

```diff
@@ -100,7 +100,7 @@ trait TLVGen {
       outcomes <- Gen.listOfN(numOutcomes, CryptoGenerators.sha256Digest)
       totalInput <-
         Gen
-          .choose(numOutcomes + 1, Long.MaxValue)
+          .choose(numOutcomes + 1, Long.MaxValue / 10000L)
           .map(Satoshis.apply)
```

Comment:
Why not use `CurrencyUnitGenerator.positiveRealistic`

Reply:
I suppose I could but this tests even more broadly as we want to make sure DLCs are resilient even to really large things (maybe even larger than the realistic bound)